addons/website/static/src/scss/website.backend.scss

```diff
@@ -1,8 +1,37 @@
 .o_dashboards {
     background-color: #ececec;
 
+    &:has(iframe[src*='plausible']) {
+        background-color: #f9fafb; // Copied from iframe.
+    }
+
     .o_website_dashboard {
         background-color: #ececec;
+
+        &:has(iframe[src*='plausible']) {
+            background-color: #f9fafb; // Copied from iframe.
+
+            iframe {
+                // TODO: Find another way to set the height, not robust enough,
+                // especially if plausible change those values later.
+                height: 2350px; // Copied from iframe.
+                top: 2px; // Do not cover eCommerce dashboard border.
+
+                @include media-breakpoint-only(md) {
+                    height: 2500px; // Copied from iframe.
+                }
+                @include media-breakpoint-only(lg) {
+                    height: 1600px; // Copied from iframe.
+                }
+                @include media-breakpoint-only(xl) {
+                    height: 1570px; // Copied from iframe.
+                }
+                @include media-breakpoint-only(xxl) {
+                    height: 1650px; // Copied from iframe.
+                }
+            }
```

Comment:
I added a TODO here, not sure if another way is possible, but this will break if plausible change their style.
I am not sure if you already investigated all the possible ways here or not. Maybe there is no other ways (js etc) due to the cross origin issue.

Reply:
I did it this way because I got a cross-origin exception as soon as I touched the `iframe`'s `document`.
So I could not follow my initial intent of doing something like:
```js
iframe.style.height = iframe.contentWindow.document.body.scrollHeight + 'px';
```
I will check again to see if there is a way in Plausible to make the cross-origin more lenient.